infra/core/host/appservice.bicep

```diff
@@ -132,6 +132,6 @@ resource applicationInsights 'Microsoft.Insights/components@2020-02-02' existing
   name: applicationInsightsName
 }
 
-output identityPrincipalId string = managedIdentity ? appService.identity.principalId : ''
+output principalId string = managedIdentity ? appService.identity.principalId : ''
```

Comment:
I stand corrected, the official name is identityPrincipalId

https://github.com/Azure/azure-dev/blob/main/templates/common/infra/bicep/core/host/appservice.bicep#L121

Please change it back.